modules/local/alevinqc.nf

```diff
@@ -1,20 +1,22 @@
 process ALEVINQC {
 
     //
-    // This module executes alevinfry QC reporting tool on alevin results
+    // This module executes alevinfry QC reporting tool on alevin-fry results
     //
 
     tag "$meta.id"
     label 'process_low'
 
-    //The alevinqc 1.14.0 container is broken, missing some libraries - thus reverting this to previous 1.12.1 version
-    conda "bioconda::bioconductor-alevinqc=1.12.1"
```

Comment:
Ideally, this would also be a module on nf-core/modules. But if you don't have time right now, we can also address this at a later point.